local_server/test/fixtures/pbmc3k-genesets.csv

```diff
@@ -0,0 +1,12 @@
+# Test fixture
+geneset_name, geneset_description, gene_symbol, gene_description
+first geneset name,,F5, a gene_description
+first geneset name,a description, NO_SUCH_GENE, non-existent gene
+first geneset name,a description, F5, duplicate gene
+first geneset name, a description, SUMO3,
+first geneset name,, SRM,
+second geneset,,RER1
+second geneset,,SIK1
+third geneset,,NO_SUCH_GENE
+fourth_geneset,fourth description,,gene intentionally missing
+fifth_dataset,,,
```

Comment:
should this be fifth_geneset?

Reply:
The exact string doesn't really matter as it is just a name - as long as the test "expects" the same, all will be fine.  But yeah, I had a brain fault when I typed it :-)  lolz.